azext_iot/operations/_mqtt.py

```diff
@@ -7,28 +7,46 @@
 import pprint
 from time import sleep
 
-from azure.cli.core.azclierror import BadRequestError
-from azext_iot.common.utility import ensure_azure_namespace_path
-from azext_iot.common.shared import DeviceAuthApiType
+from azext_iot.common.utility import check_connection_string, ensure_azure_namespace_path
 
```

Comment:
I don't see why this module should not be moved from its current location into device_messaging provider or a mqtt provider module within iothub/providers. Everything that depends on _mqtt from operations/* has already been moved right?

Reply:
sure